tests/test_dynamodb2/test_dynamodb_validation.py

```diff
@@ -42,27 +42,6 @@ def test_valid_update_expression(table):
     ).validate()
 
 
-def test_validation_of_empty_string_key_val(table):
```

Comment:
This test originally verified that a key cannot be empty. Although true, it should also never get to this point - as keys cannot be set in the first place.

An index key *can* be set, but can never be empty. This behaviour is validated in the new boto3-test, 'test_gsi_key_cannot_be_empty'.